### ISSUE TITLE
using with 2fa

### ISSUE THREAD

**Issue by mofitn3577** [steveredden/homebridge-icloud#4]
Can anyone explain to me how I can use this with 2fa it seems there is an addon or something but I am lost and have spent hours trying to figure it out. would be much appreciated.

**Comment by TheCableGuy99** [comment]
I don't think it supports it. I've also just installed it and although it alerts the phone the icon stays in the "on" state and it keeps sending the 2fa code through to the phone.
